features/steps/jobs_steps.go

```diff
@@ -71,6 +79,8 @@ func NewJobsFeature(mongoFeature *componentTest.MongoFeature) (*JobsFeature, err
 
 	f.MongoClient = mongodb
 
+	f.AuthHandler = &mock.AuthHandlerMock{}
```

Comment:
I've got concerns about mocking out an intrinsic part of a component as part of a component tests. In my view component tests should only have their external dependencies mocked. Haven't got time today to fully discuss this but I'd like to bring this up on Monday if that's OK.

Reply:
yes of course_